src/Command/Redis/TimeSeries/TSCREATERULE.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * This file is part of the Predis package.
+ *
+ * (c) 2009-2020 Daniele Alessandri
+ * (c) 2021-2023 Till Krüss
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Predis\Command\Redis\TimeSeries;
+
+use Predis\Command\Command as RedisCommand;
+
+/**
+ * @see https://redis.io/commands/ts.createrule/
+ *
+ * Create a compaction rule
+ */
+class TSCREATERULE extends RedisCommand
+{
+    public function getId()
+    {
+        return 'TS.CREATERULE';
+    }
+
+    public function setArguments(array $arguments)
+    {
+        [$sourceKey, $destKey, $aggregator, $bucketDuration] = $arguments;
+        $processedArguments = [$sourceKey, $destKey, 'AGGREGATION', $aggregator, $bucketDuration];
```

Comment:
Maybe we should add a convenience wrapper on aggregations - so that a user can get the list of potential aggregations, and also validate against them...

Reply:
I think so. Other clients have added (to json) the use case where we enable setting from a file (so json.set_file) and a directory full of files.

IMHO the clients should be low level.. but in some cases (we can definitely talk about which) helper functions become useful. This is a think in go-redis already .. basically all over the place.

Reply:
If you feel that's out of scope of this PR, then all good - it can be in other issues, clearly.